client/src/utils/protoHelpers.ts

```diff
@@ -68,22 +69,14 @@ export function protoParser(raw: Uint8Array) {
 }
 
 export function isProtoEnabled(): boolean {
-    return window.localStorage.getItem('protoEnabled') === 'true';
+    return getContextItem('protoEnabled');
```

Comment:
Man, you're gonna hate me.. but... I think we should make a hard switch to always use ProtoBuf. So, we should just hardcode `isProtoEnabled` to return `true`. I think it would remove a lot of complexity that's not adding a ton of value. What do you think?

Reply:
Do you mean to only provide proto instead of giving users the option to switch?

Reply:
And for Prometheus, do we still want to leave the option for users or a hard switch?

Reply:
Yes, only proto (no option to switch). Same for Prometheus. Let's not give them the option to decide for now.